docs/user-guide/deployments/deploy-on-kubernetes/deploy-greptimedb-cluster.md

```diff
@@ -1,99 +1,176 @@
+
+```bash
+helm install greptimedb \
+  --set meta.etcdEndpoints=etcd.etcd-cluster.svc.cluster.local:2379 \
+  --set auth.enabled=true \
+  --set auth.users[0].username=admin \
+  --set auth.users[0].password=admin \
+  greptime/greptimedb-cluster \
+  --create-namespace \
+  -n greptimedb-cluster
+```
+
+### Resource requests and limits
+To control resource allocation (CPU and memory), modify the Helm installation command as follows:
 
 ```shell
-helm install greptimedb greptime/greptimedb-cluster \
+helm install greptimedb \
   --set meta.etcdEndpoints=etcd.etcd-cluster.svc.cluster.local:2379 \
   --set meta.podTemplate.main.resources.requests.cpu=<cpu-resource> \
   --set meta.podTemplate.main.resources.requests.memory=<mem-resource> \
   --set datanode.podTemplate.main.resources.requests.cpu=<cpu-resource> \
   --set datanode.podTemplate.main.resources.requests.memory=<mem-resource> \
   --set frontend.podTemplate.main.resources.requests.cpu=<cpu-resource> \
   --set frontend.podTemplate.main.resources.requests.memory=<mem-resource> \
+  greptime/greptimedb-cluster \
   --create-namespace \
   -n greptimedb-cluster
 ```
 
-For a comprehensive list of configurable values via Helm,
-please refer to the [value configuration](https://github.com/GreptimeTeam/helm-charts/blob/main/charts/greptimedb-cluster/README.md#values).
-
-<!-- ### Use a different GreptimeDB version
-
-### Specify the database configuration file -->
-
-## Using kubectl for deployment
-
-Alternatively, you can manually create a GreptimeDB cluster using `kubectl`.
-Start by creating a configuration file named `greptimedb-cluster.yaml` with the following content:
-
-```yml
-apiVersion: greptime.io/v1alpha1
-kind: GreptimeDBCluster
-metadata:
-  name: greptimedb
-  namespace: greptimedb-cluster
-spec:
-  base:
-    main:
-      image: greptime/greptimedb
-  frontend:
-    replicas: 1
-  meta:
-    replicas: 1
-    etcdEndpoints:
-      - "etcd.etcd-cluster.svc.cluster.local:2379"
-  datanode:
-    replicas: 3
+### Complex configurations with values.yaml
+
+For more intricate configurations, first download the default `values.yaml` file and modify it locally.
+
+```bash
+curl -sLo values.yaml https://raw.githubusercontent.com/GreptimeTeam/helm-charts/main/charts/greptimedb-cluster/values.yaml
 ```
 
-Apply this configuration to instantiate the GreptimeDB cluster:
+You can configure each component by specifying custom settings in the `configData` fields. For more details, refer to the [configuration](../configuration.md) documentation. 
+
+Below is an example of how to modify the settings in the `values.yaml` file. This example demonstrates how to configure specific components within GreptimeDB. It sets the metasrv's selector type to `round_robin`, adjusts the datanode’s Mito engine by setting `global_write_buffer_size` to `4GB`, and configures the frontend's meta client `ddl_timeout` to `60s`:
```

Comment:
The example should use common configurations.
Is these common configurations that users need to be concerned about?